src/helpers/terminal_helpers.sh

```diff
@@ -181,6 +181,13 @@ function run_command {
 }
 
 
+# Run a command silently (won't print to console)
+function run_command_silently {
```

Comment:
That doesn't work. We are expecting the command to be all the arguments (though I'm uncertain if thats still needed). Maybe we can have the command be the first argument and silent be the second (optional)?